ema_workbench/examples/regional_sa_flu.py

```diff
@@ -0,0 +1,22 @@
+""" A simple example of performing regional sensitivity analysis
+
+
+"""
+import matplotlib.pyplot as plt
+
+from ema_workbench.analysis import regional_sa
+from ema_workbench import ema_logging, load_results
+
+fn = "./data/1000 flu cases with policies.tar.gz"
+results = load_results(fn)
+x, outcomes = results
+
+y = outcomes["deceased population region 1"][:, -1] > 1000000
+
+fig = regional_sa.plot_cdfs(x, y)
+# regional_sa.plot_cdfs(x, y, ccdf=True)
```

Comment:
Why are they commented out? Maybe add comment explaining that.

Reply:
just copy paste from test code. Will remove them.